frontend/src/components/assessments/assessment-creation/AssessmentQuestions.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Button, HStack, Spacer, Text, VStack } from "@chakra-ui/react";
+
+import { PlusOutlineIcon } from "../../../assets/icons";
+import { CREATE_QUESTION } from "../../../constants/Routes";
+import { QuestionData } from "../../../constants/TestConstants";
+
+import AddQuestionCard from "./AddQuestionCard";
+import QuestionCard from "./QuestionCard";
+import QuestionSummary from "./QuestionSummary";
+
+interface AssessmentQuestionsProps {
+  questions: QuestionData[];
+}
```

Comment:
I think the type used here will probably change when i create the pr to save data from the question editor